docs/.vuepress/components/TwoColumnGrid.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="two-column__wrap">
+    <cdr-grid class="two-column__grid">
+      <div
+        v-for="(example, idx) in examples"
+        :key="`${example.type}${idx}`"
+      >
+        <figure class="two-column">
+          <cdr-img
+            class="two-column__image"
+            :src="$withBase(`/${example.image}`)"
+            :ratio="getRatio(example.ratio) || '16-9'"
+            :alt="example.alt || ''" />
+          <figcaption class="two-column__caption">
+            <span>{{ example.caption }}</span>
+            <slot/>
+          </figcaption>
+        </figure>
+      </div>
+    </cdr-grid>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "TwoColumnGrid",
```

Comment:
I think the component name should a little more specific to what this component is actually doing, there's probably going to be a couple different components that render content in a two-col grid